src/Timer_main.cpp

```diff
@@ -0,0 +1,451 @@
+        {
+            DBGLN("Enter binding mode...");
+            bindingStart =  now;
+            connectionState = binding;
+            isBinding = true;
+        }
+        else if (packet->payload[0] == 'W')
+        {
+            DBGLN("Enter WIFI mode...");
+            connectionState = wifiUpdate;
+            devicesTriggerEvent();
+        }
+        SendInProgressResponse();
+      }
+    }
+  case MSP_ELRS_GET_BACKPACK_VERSION:
+    DBGLN("Processing MSP_ELRS_GET_BACKPACK_VERSION...");
+    SendVersionResponse();
+    break;
+  case MSP_ELRS_SET_SEND_UID:
+  DBGLN("Processing MSP_ELRS_SET_SEND_UID...");
+    {
+      uint8_t function = packet->readByte();
+
+      // Set target Send address
+      if (function == 0x0001)
+      {
+        uint8_t receivedAddress[6];
+        receivedAddress[0] = packet->readByte();
+        receivedAddress[1] = packet->readByte();
+        receivedAddress[2] = packet->readByte();
+        receivedAddress[3] = packet->readByte();
+        receivedAddress[4] = packet->readByte();
+        receivedAddress[5] = packet->readByte();
+
+        // Register peer address
+        memset(&peerInfo, 0, sizeof(peerInfo));
+        memcpy(peerInfo.peer_addr, receivedAddress, 6);
+        peerInfo.channel = 0;
+        peerInfo.encrypt = false;
+        esp_now_add_peer(&peerInfo);
+
+        // Set Send address for new target
+        memset(&sendAddress, 0, sizeof(sendAddress));
+        memcpy(sendAddress, receivedAddress, 6);
+      }
+
+      // Return to normal for recieving messages
+      else
+      {
+        // Unregister Peer
+        esp_now_del_peer(sendAddress);
+
+        // Set Send address for normal target
+        memset(&sendAddress, 0, sizeof(sendAddress));
+        memcpy(sendAddress, firmwareOptions.uid, 6);
+      }
+
+      // Configure MAC address
+      #if defined(PLATFORM_ESP8266)
+        wifi_set_macaddr(STATION_IF, sendAddress);
+      #elif defined(PLATFORM_ESP32)
+        esp_wifi_set_mac(WIFI_IF_STA, sendAddress);
+      #endif
+      break;
+    }
+  default:
+    // transparently forward MSP packets via espnow to any subscribers
+    sendMSPViaEspnow(packet);
+    break;
+  }
+}
+
+int sendMSPViaEspnow(mspPacket_t *packet)
+{
+  int esp_err = -1;
+  uint8_t packetSize = msp.getTotalPacketSize(packet);
+  uint8_t nowDataOutput[packetSize];
+
+  uint8_t result = msp.convertToByteArray(packet, nowDataOutput);
+
+  if (!result)
+  {
+    // packet could not be converted to array, bail out
+    return esp_err;
+  }
+
+  esp_err = esp_now_send(sendAddress, (uint8_t *) &nowDataOutput, packetSize);
+
+  blinkLED();
+  return esp_err;
+}
+
+void SetSoftMACAddress()
+{
+  DBGLN("EEPROM MAC = ");
+  for (int i = 0; i < 6; i++)
+  {
+    #ifndef MY_UID
+    memcpy(firmwareOptions.uid, config.GetGroupAddress(), 6);
+    #endif
+    DBG("%x", firmwareOptions.uid[i]); // Debug prints
+    DBG(",");
+  }
+  DBGLN(""); // Extra line for serial output readability
```

Comment:
Change this section to this
```cpp
  if (!firmwareOptions.hasUID)
  {
    memcpy(firmwareOptions.uid, config.GetGroupAddress(), 6);
  }
  DBG("EEPROM MAC = ");
  for (int i = 0; i < 6; i++)
  {
    DBG("%x", firmwareOptions.uid[i]); // Debug prints
    DBG(",");
  }
  DBGLN(""); // Extra line for serial output readability
```

Reply:
I'm assuming theres a bunch of messages that are not really required to be handled in this instance like Bind mode and Wifi mode commands.